_includes/docs-info.html

```diff
@@ -42,17 +42,19 @@ <h6>For Users</h6>
         <a name="tutorials"></a>
         <h5>Tutorials for the finer steps</h5>
             <div class="row docs-details">
-                <a href="https://bit.ly/2Fyn97v">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated July 21st 2017.
+                <a href="https://bit.ly/3q73CTo">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2WsFiun">Study-specific shape analysis (GROUPS)</a> Last updated July 16th 2018.
+                <a href="https://bit.ly/3q6v1oo">Study-specific shape analysis (GROUPS)</a> Last updated September 2022.
```

Comment:
https://docs.google.com/presentation/d/1n2-138ZsspF2iL4qF3DTrcHrSV9YEgTdAVEOwqVRK7s/present?slide=id.p48

Should the link to the Github GROUPS repo point to NIRALUser instead of MahmoudMostapha?

Reply:
This was updated to point to NIRALUser.